scripts/generateAlias.ts

```
@@ -0,0 +1,61 @@
+import { readdirSync, statSync, writeFileSync } from 'fs'
+import { join } from 'path'
+
+const GENERATED_PATH = 'packages_generated'
+const OUTPUT_PATH = 'packages/sdk/src/index.ts'
+
+const toPascal = (s: string) =>
+  s
+    .split(/[_-]/)
+    .map(part => part.charAt(0).toUpperCase() + part.slice(1))
+    .join('')
+
+const toSlug = (s: string) => s.replace(/_/g, '-')
+
+const services = readdirSync(GENERATED_PATH).filter(folder => {
+  const fullPath = join(GENERATED_PATH, folder)
+
+  return statSync(fullPath).isDirectory()
+})
+
+let output = `/* Auto-generated. Do not edit manually. */\n\n`
+
+for (const service of services) {
+  const slug = toSlug(service) // ex. "audit-trail"
+  const pascal = toPascal(service) // ex. "AuditTrail"
+  const srcPath = join(GENERATED_PATH, service, 'src')
+
+  let versions: string[] = []
+  try {
+    versions = readdirSync(srcPath).filter(vFolder => {
+      const vPath = join(srcPath, vFolder)
+
+      return statSync(vPath).isDirectory() && /^v[0-9a-z]+$/i.test(vFolder)
+    })
+  } catch {
+    console.warn(`⚠️  Pas de dossier 'src' pour ${service}, ignoré.`)
+    // eslint-disable-next-line no-continue
+    continue
```

same

---

i think we should throw an error instead